src/conf/gunicorn_backbone.py

```diff
@@ -0,0 +1,21 @@
+"""Gunicorn config."""
+
+import logging
+from src.settings import GUNICORN_SETTINGS
+
+workers = GUNICORN_SETTINGS.workers
+worker_class = GUNICORN_SETTINGS.worker_class
+timeout = GUNICORN_SETTINGS.timeout
+preload_app = GUNICORN_SETTINGS.preload
+reload = preload_app is not True
+
+
+def when_ready(server):  # noqa
```

Comment:
It already there part of gunicorn's INFO print.
```
[2021-01-13 09:53:19,120] INFO in /src/conf/gunicorn_backbone.py:25: Starting backbone gunicorn with 1 workers gevent worker class and preload True
[2021-01-13 09:53:19 +0000] [12] [INFO] Booting worker with pid: 12
[2021-01-13 09:53:32 +0000] [12] [INFO] Worker exiting (pid: 12)
[2021-01-13 09:53:32 +0000] [27] [INFO] Booting worker with pid: 27
```

It was missing earlier because we weren't passing `--log-level` to gunicorn.